### PR TITLE
[IT] Add `Upgrading to lazer`

### PR DESCRIPTION
**Pull request by Akahito0**
Italian translation

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by Sclafus** [comment]
en has been deleted, might be an error while renaming the file.

**Comment by Akahito0** [comment]
for some reason i can't remove the request for a review, sorry about this

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+
+Puoi trovare il download [Qui](https://github.com/ppy/osu#running-osu). Prossimamente, sarai in grado di passare a lazer direttamente da stable (dall'impostazione `Release stream`) e' trovare il link di download nel sito di osu!.
+
+## FAQ
+
+### Migrazione
+
+#### Stable se ne andrà via? Sono forzato a stare su Lazer?
+
+Stable continuerà a essere mantenuto finche' gli utenti lo utilizzeranno. Perlomeno, sara' mantenuto per diversi anni.
```
Comment (GYGY88) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:105]: "verrá" is a bit better than "sará"

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+
+gli score verranno mostrati sotto la sezione "Partite Recenti" ma non in "Migliore Performance" per ora.
+
+#### Se imposto un punteggio su lazer, darà punti performance?
+
+I punteggi avranno gia' i punti performance calcolati (puoi vederlo nella sezione "Partite recenti" del tuo profilo), ma non contribuiranno ancora al valore totale.
+
+#### Lazer usa ScoreV2?
+
+Lazer attualmente utilizza un nuovo sistema di punteggio sperimentale che e' simile a ScoreV2 ma non e' identico. questo e' ancora sotto sviluppo e stiamo raccogliendo pareri dai player in base a quanto bene va in vari contesti (solo play, classifiche, tornei etc.)
```
Comment (GYGY88) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:147]: you can translate "player" into "giocatore" imo

Comment (GYGY88) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:147]: "quanto bene va" doesn't seem appropriate for a wiki context. i'd rephrase it in a more formal way "...stiamo ancora raccogliendo pareri e opinioni dai giocatori in base al contesto in cui viene usato (solo play, classifiche, tornei etc.).

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+
+#### Se imposto un punteggio su lazer, rimarrà per sempre?
+
+Anche se cercheremo di preservare il maggior numero possibile di punteggi, **non garantiamo che i punteggi rimarranno a tempo indeterminato**. In qualsiasi momento potremmo decidere di cancellare un sottoinsieme o tutti i punteggi per preservare l'equilibrio del gioco.
+
+#### I punteggi impostati su stable saranno visualizzati in lazer?
+
+Sì. Una volta terminato il bilanciamento della combinazione di punteggi in lazer e in stable, entrambi saranno visibili.
+
+#### Tutte le mod saranno Classificato?
```
Comment (GYGY88) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:163]: plural on "Classificato"

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+
+Per ora, i punteggi di tutte le combinazioni di mod appaiono nelle classifiche. Si sta ancora discutendo se i punteggi daranno punti alle prestazioni con tutte le mod (e se lo faranno, se ci sarà un bonus o una penalità).
```
Comment (GYGY88) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:165]: adapt "punti alle prestazioni" into "punti performance"

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 5]
```diff
@@ -0,0 +1,284 @@
...
+
+#### Non mi piacciono le nuove meccaniche di gioco. Posso ripristinare le vecchie meccaniche di gioco come su stable?
+
+Prova ad applicare la mod "classic", che ripristinera' gran parte del vecchio comportamento a cui sei abituato. Assicurati anche di controllare le impostazioni offerte dalla mod classica, in quanto ti consentiranno di personalizzare ulteriormente la tua esperienza e di capire quali modifiche vengono applicate (in quanto sono tutte elencate).
```
Comment (GYGY88) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:169]: "vecchio comportamento" isn't a very optimal adaptation. "...delle vecchie meccaniche di gioco..." seems better

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 6]
```diff
@@ -0,0 +1,284 @@
...
+
+### Skinning e UI
+
+#### Qualcosa si comporta in modo diverso rispetto a stable e non mi piace!
+
+Eseguire la configurazione guidata nella parte superiore delle impostazioni e controllare le impostazioni nella schermata `Comportamento`. Qui sono elencate molte delle impostazioni comuni che sono state modificate per impostazione predefinita. C'è anche un singolo pulsante che si può premere per applicare i vecchi comportamenti come punto di partenza per il vostro viaggio con il lazer.
```
Comment (GYGY88) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:175]: we don't really know how "behaviour" is adapted in the italian version of the game so i'm not sure if "comportamento" is the right choice for now

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 7]
```diff
@@ -0,0 +1,284 @@
...
+
+#### le vecchie skin funzioneranno nella selezione delle mappe e nella schermata del risultato?
+
+Faremo del nostro meglio per riportare queste funzionalità 
+
+#### posso usare il cursore della skin pure sul menu'?
+
+porteremo molto probabilmente questa novita' date la richiesta elevata.
```
Comment (GYGY88) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:183]: Funzionalitá

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 8]
```diff
@@ -0,0 +1,284 @@
...
+
+Alcuni filtri nell'elenco delle beatmap sono ancora riservati ai sostenitori.
+
+Ci sono anche alcuni vantaggi aggiuntivi:
+
+- I supporter possono creare playlist che durano più a lungo.
+
+Intendiamo valutare nuovi vantaggi in futuro, ma al momento ci concentriamo sulla parità di funzionalità con la stable, quindi vi invitiamo a utilizzare l'acquisto del tag Supporter come un modo per... sostenere lo sviluppo del gioco!
+
+#### Se baro su lazer verrò bannato?
```
Comment (GYGY88) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:241]: "barare" is more commonly used in board games or similar i think that "fare uso di trucchi" makes the cheating topic more clear for the common user

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 9]
```diff
@@ -0,0 +1,284 @@
...
+
+#### Manca una funzione da cui dipendo! / Qualcosa è cambiato e non mi piace. / Ho trovato un bug, qual è il modo migliore per segnalarlo?
+
+È molto probabile che ne siamo già a conoscenza e che lo stiamo monitorando per una futura implementazione! Cerca nell'[issue tracker](https://github.com/ppy/osu/issues) e nella [pagina delle discussioni](https://github.com/ppy/osu/discussions). Se non si riesce a trovare nessuna discussione corrispondente, sentirsi liberi di [aprire una discussione](https://github.com/ppy/osu/discussions/new).
+
+Si noti che stiamo già monitorando più di 1.000 problemi di varia priorità e che potrebbe volerci del tempo per risolvere problemi che riguardano solo un piccolo numero di utenti.
+
+### Altro
+
+#### Perché ci vuole così tanto tempo per diventare la versione "principale"?
```
Comment (GYGY88) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:213]: i think that we can't make the subject implicit in this sentence it would be better to rephrase it to something like "Perché lazer ci sta mettendo così tanto tempo per diventare la versione "principale"?"

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 10]
```diff
@@ -0,0 +1,284 @@
...
+
+porteremo molto probabilmente questa novita' date la richiesta elevata.
+
+### Prestazioni
+
+#### perche' non posso avere gli fps illimitati?
+
+Al di sopra di una certa soglia, non c'è motivo di utilizzare frame rate più elevati. Lazer impiega diverse nuove tecnologie per garantire la latenza più bassa possibile senza richiedere frame rate elevati. Questo aspetto continuerà a migliorare in futuro, poiché abbiamo ancora ulteriori miglioramenti da implementare.
+
+Lazer esegue il polling per l'ingresso a 1000 Hz indipendentemente dal limitatore di FPS, motivo per cui anche l'impostazione massima del limitatore limiterà a 1000 FPS.
```
Comment (GYGY88) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:191]: you can write "polling rate in ingresso" instead of "il polling per l'ingresso"

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 11]
```diff
@@ -0,0 +1,284 @@
...
+
+#### Lazer non funziona in modo stabile. Cosa succede?
+
+Sebbene sulla maggior parte dell'hardware moderno notiamo che lazer supera stable, però, ci sono sempre casi dove utenti non hanno una configurazione hardware performante. Nella nostra roadmap a breve termine, stiamo cercando di supportare DirectX (alias "modalità compatibile" su stable) e Vulkan, che hanno entrambi un supporto driver migliore di OpenGL su tutto l'hardware. Una volta implementato, le prestazioni su hardware come i chipset integrati Intel miglioreranno notevolmente.
+
+### Fornire feedback
+
+#### Manca una funzione da cui dipendo! / Qualcosa è cambiato e non mi piace. / Ho trovato un bug, qual è il modo migliore per segnalarlo?
+
+È molto probabile che ne siamo già a conoscenza e che lo stiamo monitorando per una futura implementazione! Cerca nell'[issue tracker](https://github.com/ppy/osu/issues) e nella [pagina delle discussioni](https://github.com/ppy/osu/discussions). Se non si riesce a trovare nessuna discussione corrispondente, sentirsi liberi di [aprire una discussione](https://github.com/ppy/osu/discussions/new).
```
Comment (GYGY88) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:207]: since you are directly talking to a person i think that it's better to write "sentitevi" instead of "sentirsi"

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 12]
```diff
@@ -0,0 +1,284 @@
...
+
+Sì.
+
+#### Se trovo qualcuno che bara su lazer, come devo segnalarlo?
+
+Nello stesso modo in cui lo faresti di solito.
+
+#### Dove sono le microtransazioni?
+
+Probabilmente stai pensando a un altro gioco.
```
Comment (GYGY88) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:251]: "ad" instead of "a"

**Review by GYGY88 — CHANGES_REQUESTED** [review]

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 13]
```diff
@@ -0,0 +1,284 @@
...
+
+No.
+
+#### se disinstallo stable, il contenuto importato da stable darà problemi su lazer?
+
+No.
+
+#### Se installo lazer, sarò in grado di ritornare su stable?
+
+Si, lazer verrà sempre installato a fianco a stable. A meno che tu non decida di eliminare uno dei due, Entrambi saranno accessibili.
```
Comment (Sclafus) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:127]: "Entrambi" should be lower-case

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 14]
```diff
@@ -0,0 +1,284 @@
...
+
+#### Lazer usa ScoreV2?
+
+Lazer attualmente utilizza un nuovo sistema di punteggio sperimentale che è simile a ScoreV2 ma non è identico. questo è ancora sotto sviluppo e sstiamo ancora raccogliendo pareri e opinioni dai giocatori in base al contesto in cui viene usato (solo play, classifiche, tornei etc.).
+
+<!-- lint ignore no-heading-punctuation -->
+
+#### preferisco il vecchio sistema di punteggio, dove il punteggio diventa veramente grande.
+
+Puoi effettivamente modificare l'impostazione `Modalità di visualizzazione del punteggio` in `Classico` per ripristinare lo stile esplosivo del punteggio in tutto il gioco! Non sarà un abbinamento perfetto, ma ti darà la stessa sensazione del punteggio classico e verrà applicato ovunque ti aspetteresti che sia.
```
Comment (Sclafus) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:153]: "abbinamento perfetto" doesn't work in this context.

Comment (Sclafus) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:153]: "verrà applicato ovunque ti aspetteresti che sia" -> "verrà applicato dove ti aspetteresti"

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 15]
```diff
@@ -0,0 +1,284 @@
...
+
+#### Se imposto un punteggio su lazer, rimarrà per sempre?
+
+Anche se cercheremo di preservare il maggior numero possibile di punteggi, **non garantiamo che i punteggi rimarranno a tempo indeterminato**. In qualsiasi momento potremmo decidere di cancellare un sottoinsieme o tutti i punteggi per preservare l'equilibrio del gioco.
+
+#### I punteggi impostati su stable saranno visualizzati in lazer?
+
+Sì. Una volta terminato il bilanciamento della combinazione di punteggi in lazer e in stable, entrambi saranno visibili.
+
+#### Tutte le mod saranno Classificate?
```
Comment (Sclafus) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:163]: capital c

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 16]
```diff
@@ -0,0 +1,284 @@
...
+
+#### posso usare il cursore della skin pure sul menù?
+
+porteremo molto probabilmente questa novità date la richiesta elevata.
+
+### Prestazioni
+
+#### perchè non posso avere gli fps illimitati?
+
+Al di sopra di una certa soglia, non c'è motivo di utilizzare frame rate più elevati. Lazer impiega diverse nuove tecnologie per garantire la latenza più bassa possibile senza richiedere frame rate elevati. Questo aspetto continuerà a migliorare in futuro, poiché abbiamo ancora ulteriori miglioramenti da implementare.
```
Comment (Sclafus) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:189]: "ancora ulteriori miglioramenti" -> "altre migliorie"

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 17]
```diff
@@ -0,0 +1,284 @@
...
+
+Lazer esegue il polling rate in ingresso a 1000 Hz indipendentemente dal limitatore di FPS, motivo per cui anche l'impostazione massima del limitatore limiterà a 1000 FPS.
```
Comment (Sclafus) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:191]: This doesn't sound right.
"Lazer prende l'input a 1000Hz indipendentemente dal limitatore di FPS, che è anche il motivo per cui l'impostazione massima del limitatore è 1000 FPS."

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 18]
```diff
@@ -0,0 +1,284 @@
...
+
+Se siete curiosi di sapere come questo influisce sulla latenza in ingresso e di verificare la vostra percezione, eseguite il "certificatore di latenza" integrato in fondo alle impostazioni.
```
Comment (Sclafus) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:193]: "Latency certifier" is not yet translated to italian in lazer. I suggest using the actual in-game name, which is: "Run latency certifier"

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 19]
```diff
@@ -0,0 +1,284 @@
...
+
+#### Se il polling dell'input viene eseguito solo a 1000 Hz, che ne è del mio mouse da gioco a 8000 Hz?
+
+Il sistema operativo continuerà a eseguire il polling alla frequenza più alta possibile, anche se i vantaggi sono dimostrati essere irrilevanti. Il polling a velocità così elevate può comportare richiesta di potenza aggiuntiva imprevista e si consiglia di limitare le proprie periferiche a 1000 Hz per garantire una esperienza stabile.
```
Comment (Sclafus) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:197]: "sono dimostrati essere irrilevanti" -> " sono trascurabili"

Comment (Sclafus) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:197]: "imprevista" seems out of place. I suggest removing it completely.

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 20]
```diff
@@ -0,0 +1,284 @@
...
+
+#### Lazer non funziona in modo stabile. Cosa succede?
```
Comment (Sclafus) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:199]: Doesn't look like the original.
I suggest: "Lazer performa peggio di stable per me. Che succede?"

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 21]
```diff
@@ -0,0 +1,284 @@
...
+
+Sebbene sulla maggior parte dell'hardware moderno notiamo che lazer supera stable, però, ci sono sempre casi dove utenti non hanno una configurazione hardware performante. Nella nostra roadmap a breve termine, stiamo cercando di supportare DirectX (alias "modalità compatibile" su stable) e Vulkan, che hanno entrambi un supporto driver migliore di OpenGL su tutto l'hardware. Una volta implementato, le prestazioni su hardware come i chipset integrati Intel miglioreranno notevolmente.
+
+### Fornire feedback
+
+#### Manca una funzione da cui dipendo! / Qualcosa è cambiato e non mi piace. / Ho trovato un bug, qual è il modo migliore per segnalarlo?
+
+È molto probabile che ne siamo già a conoscenza e che lo stiamo monitorando per una futura implementazione! Cerca nell'[issue tracker](https://github.com/ppy/osu/issues) e nella [pagina delle discussioni](https://github.com/ppy/osu/discussions). Se non si riesce a trovare nessuna discussione corrispondente, sentitevi liberi di [aprire una discussione](https://github.com/ppy/osu/discussions/new).
```
Comment (Sclafus) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:207]: I would rework the phrase
"Se non trovate nessuna discussione che corrisponde al vostro problema, sentitevi liberi di [aprirne una nuova](https://github.com/ppy/osu/discussions/new)"

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 22]
```diff
@@ -0,0 +1,284 @@
...
+
+Si noti che stiamo già monitorando più di 1.000 problemi di varia priorità e che potrebbe volerci del tempo per risolvere problemi che riguardano solo un piccolo numero di utenti.
```
Comment (Sclafus) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:209]: "di varia priorità" -> "più o meno critici"

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 23]
```diff
@@ -0,0 +1,284 @@
...
+
+Infine, a differenza dell'ultima iterazione, stiamo dedicando tempo e diligenza per garantire che la base del codice sia utile anche in futuro. Abbiamo fatto tutto questo lavoro per consentire alle nuove funzionalità di essere online a velocità incredibile. Questo includerà nuovi componenti dell'interfaccia utente, nuovi modi di personalizzare il gioco, nuovi sistemi multiplayer e, non dimentichiamolo, la possibilità di caricare e giocare tutte le beatmap esistenti su modalità di gioco (alias, ruleset personalizzati) completamente nuove!
+
+#### Cosa ci aspetta?
+
+Abbiamo un'enorme quantità di funzioni e miglioramenti richiesti dagli utenti che continueremo a distribuire alla velocità della luce. Per coloro che si sono uniti a noi di recente e che non hanno ancora sperimentato lo slancio dello sviluppo di osu!
```
Comment (Sclafus) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:223]: "miglioramenti" -> "migliorie"

Comment (Sclafus) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:223]: It's missing a part of the original phrase.

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 24]
```diff
@@ -0,0 +1,284 @@
...
+
+#### Come si accede alla cartella delle canzoni?
+
+In lazer non esiste una cartella delle canzoni! Questo ci permette di fare cose interessanti, come non dover premere `F5` alla selezione della mappa per aggiornare le beatmap (perché le beatmap sono sempre in stato ottimale) e di ridurre lo spazio su disco utilizzato dalle beatmap del 20-40%. È possibile leggere ulteriori informazioni su [il modo in cui lazer memorizza i file] (/wiki/Client/Release_stream/Lazer/File_storage).
+
+Se è necessario apportare modifiche a una beatmap, utilizzare l'editor. In futuro introdurremo una modalità nell'editor che renderà temporaneamente accessibile la cartella di una beatmap per modifiche esterne. Ciò consentirà di utilizzare strumenti esterni su una beatmap durante il processo di creazione.
```
Comment (Sclafus) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:229]: "utilizzare l'editor" -> "vi consigliamo di utilizzare l'editor"

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 25]
```diff
@@ -0,0 +1,284 @@
...
+
+#### Ora che "osu!direct" è disponibile per tutti i giocatori, i sostenitori avranno nuovi vantaggi?
+
+Alcuni filtri nell'elenco delle beatmap sono ancora riservati ai sostenitori.
+
+Ci sono anche alcuni vantaggi aggiuntivi:
+
+- I supporter possono creare playlist che durano più a lungo.
+
+Intendiamo valutare nuovi vantaggi in futuro, ma al momento ci concentriamo sulla parità di funzionalità con la stable, quindi vi invitiamo a utilizzare l'acquisto del tag Supporter come un modo per... sostenere lo sviluppo del gioco!
```
Comment (Sclafus) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:239]: "la stable" -> "stable"

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 26]
```diff
@@ -0,0 +1,284 @@
...
+
+Abbiamo un'enorme quantità di funzioni e miglioramenti richiesti dagli utenti che continueremo a distribuire alla velocità della luce. Per coloro che si sono uniti a noi di recente e che non hanno ancora sperimentato lo slancio dello sviluppo di osu!
+
+#### Come si accede alla cartella delle canzoni?
+
+In lazer non esiste una cartella delle canzoni! Questo ci permette di fare cose interessanti, come non dover premere `F5` alla selezione della mappa per aggiornare le beatmap (perché le beatmap sono sempre in stato ottimale) e di ridurre lo spazio su disco utilizzato dalle beatmap del 20-40%. È possibile leggere ulteriori informazioni su [il modo in cui lazer memorizza i file] (/wiki/Client/Release_stream/Lazer/File_storage).
+
+Se è necessario apportare modifiche a una beatmap, utilizzare l'editor. In futuro introdurremo una modalità nell'editor che renderà temporaneamente accessibile la cartella di una beatmap per modifiche esterne. Ciò consentirà di utilizzare strumenti esterni su una beatmap durante il processo di creazione.
+
+#### Ora che "osu!direct" è disponibile per tutti i giocatori, i sostenitori avranno nuovi vantaggi?
```
Comment (Sclafus) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:231]: "supporter" is not translated in the osu website.

**Review by Sclafus — CHANGES_REQUESTED** [review]
Capitalization is inconsistent and some sections are not translated.
Also, "Funzionalitá" should be "Funzionalità", with the other accent.

**Comment by The-Last-Cookie** [comment]
Please don't forget to apply the changes from #8548.

**Comment by Akahito0** [comment]
Will do !

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 27]
```diff
@@ -192,9 +193,11 @@ Lazer prende l'input a 1000Hz indipendentemente dal limitatore di FPS, che è an
 
 Se siete curiosi di sapere come questo influisce sulla latenza in ingresso e di verificare la vostra percezione, eseguite il "Run latency certifier" integrato in fondo alle impostazioni.
 
-#### Se il polling dell'input viene eseguito solo a 1000 Hz, che ne è del mio mouse da gioco a 8000 Hz?
+You can also [read this technical document](https://github.com/ppy/osu/wiki/Latency-and-unlimited-frame-rates) explaining the path we are taking along with rationale behind it.
```
Comment (Sclafus) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:196]: this part is not translated

**Review by Sclafus — CHANGES_REQUESTED** [review]

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 28]
```diff
@@ -193,7 +193,7 @@ Lazer prende l'input a 1000Hz indipendentemente dal limitatore di FPS, che è an
 
 Se siete curiosi di sapere come questo influisce sulla latenza in ingresso e di verificare la vostra percezione, eseguite il "Run latency certifier" integrato in fondo alle impostazioni.
 
-You can also [read this technical document](https://github.com/ppy/osu/wiki/Latency-and-unlimited-frame-rates) explaining the path we are taking along with rationale behind it.
+Potresti anche [leggere questo documento tecnico](https://github.com/ppy/osu/wiki/Latency-and-unlimited-frame-rates) spiegando il percorso che stiamo intraprendendo e le motivazioni dietro a questa decisione.
```
Comment (Sclafus) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:196]: I don't think it's clear enough
I propose:
"Potresti anche [leggere questo documento tecnico](https://github.com/ppy/osu/wiki/Latency-and-unlimited-frame-rates) in cui viene spiegato il percorso che stiamo intraprendendo e le motivazioni dietro alle decisioni prese."

**Review by Sclafus — APPROVED** [review]
LGTM

**Review by Sclafus — APPROVED** [review]
LGTM

**Review by Walavouchey — CHANGES_REQUESTED** [review]
many sentences (both in headings and paragraph text) start in lowercase - that's not intended right?

**Comment by Akahito0** [comment]
Right, it's not intended, will proceed to fix now

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 29]
```diff
@@ -0,0 +1,290 @@
...
+
+Nello stesso modo in cui lo faresti di solito.
+
+#### Dove sono le microtransazioni?
+
+Probabilmente stai pensando ad un altro gioco.
+
+### Notes
+
+[^wine]: usando wine.
```
Comment (Walavouchey) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:260]:
```suggestion
[^wine]: Usando wine.
```

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 30]
```diff
@@ -0,0 +1,290 @@
...
+[^compatibility-mode]: DirectX in modalità compatibilità.
+[^coming-soon]: arriva presto.
```
Comment (Walavouchey) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:262]:
```suggestion
[^coming-soon]: Arriva presto.
```

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 31]
```diff
@@ -0,0 +1,290 @@
...
+
+## Differenze tra stable e lazer
+
+Di seguito verrà rappresentata una lista dello **stato attuale** di lazer in confronto con stable. Bisogna tenere a mente che il progetto è in continua evoluzione — l'obiettivo finale è quello di implementare tutte le funzioni che i giocatori hanno richiesto nel corso degli anni.
+
+### Compatibilità e prestazioni
+
+| Funzionalità | stable | lazer |
+| :-- | :-- | :-- |
+| Windows 8.0 e versioni precedenti | ![Yes][true] | ![No][false] |
```
Comment (Walavouchey) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:24]: the Yes and No and Partial should be translated

they appear if the image can't be loaded

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 32]
```diff
@@ -0,0 +1,290 @@
...
+
+### Fornire feedback
+
+#### Manca una funzione da cui dipendo! / Qualcosa è cambiato e non mi piace. / Ho trovato un bug, qual è il modo migliore per segnalarlo?
+
+È molto probabile che ne siamo già a conoscenza e che lo stiamo monitorando per una futura implementazione! Cerca nell'[issue tracker](https://github.com/ppy/osu/issues) e nella [pagina delle discussioni](https://github.com/ppy/osu/discussions). Se non trovate nessuna discussione che corrisponde al vostro problema, sentitevi liberi di [aprirne una nuova](https://github.com/ppy/osu/discussions/new).
+
+Si noti che stiamo già monitorando più di 1.000 problemi più o meno critici e che potrebbe volerci del tempo per risolvere problemi che riguardano solo un piccolo numero di utenti.
+
+### Altro
```
Comment (Walavouchey) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:216]: this is missing the "Why is it called "lazer"?" question. see the [french version](https://github.com/ppy/osu-wiki/pull/8481/files#diff-9cffabcd9c5de51fe9f9981448dc8188941dbb4dd8cff7fb724bc53d9a42647fR217-R219) for an example on how it could be translated

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 33]
```diff
@@ -0,0 +1,290 @@
...
+
+#### Cosa ci aspetta?
+
+Abbiamo un'enorme quantità di funzioni e migliorie richiesti dagli utenti che continueremo a distribuire alla velocità della luce. Per coloro che si sono uniti a noi di recente e che non hanno ancora sperimentato il frutto dello sviluppo di osu!, preparati per una sorpresa.
+
+#### Come si accede alla cartella delle canzoni?
+
+In lazer non esiste una cartella delle canzoni! Questo ci permette di fare cose interessanti, come non dover premere `F5` alla selezione della mappa per aggiornare le beatmap (perché le beatmap sono sempre in stato ottimale) e di ridurre lo spazio su disco utilizzato dalle beatmap del 20-40%. È possibile leggere ulteriori informazioni su [il modo in cui lazer memorizza i file] (/wiki/Client/Release_stream/Lazer/File_storage).
```
Comment (Walavouchey) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:232]:
```suggestion
In lazer non esiste una cartella delle canzoni! Questo ci permette di fare cose interessanti, come non dover premere `F5` alla selezione della mappa per aggiornare le beatmap (perché le beatmap sono sempre in stato ottimale) e di ridurre lo spazio su disco utilizzato dalle beatmap del 20-40%. È possibile leggere ulteriori informazioni su [il modo in cui lazer memorizza i file](/wiki/Client/Release_stream/Lazer/File_storage).
```
